docs/generateAllConfigPartial.java

```diff
@@ -0,0 +1,310 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.jboss:jandex:2.4.3.Final
+
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.Index;
+import org.jboss.jandex.IndexReader;
+import org.jboss.jandex.Main;
+import org.jboss.jandex.Type;
+
+import java.io.*;
+import java.util.*;
+import java.net.*;
+import java.nio.charset.*;
+import java.nio.file.*;
+import java.util.stream.Collectors;
+
+import static java.lang.System.*;
+
+public class generateAllConfigPartial {
```

Comment:
Is this something we could share across projects?

Reply:
Probably, I'll look into adding it into common app components.